src/vec_int64_ppc.h

```diff
@@ -1871,6 +3026,100 @@ vec_swapd (vui64_t vra)
   return (result);
 }
 
+/** \brief Vector Pack Unsigned Doubleword Unsigned Modulo.
+ *
+ *  The doubleword source is the concatination of vra and vrb.
+ *  For each integer word from 0 to 3, of the result vector, do the
+ *  following. The contents of bits 32:63 of the corresponding
```

Comment:
...do the following:  place the contents of bits 32:63 of the corresponding doubleword source element [i] into word element [i] of the result.

Reply:
OK